custom_tvm_ops/topi/topi.cc

```diff
@@ -0,0 +1,47 @@
+#include <tvm/build_module.h>
```

Comment:
can all of these files live in the same directory? maybe `torch_tvm/ops/layer_norm/`?  Since this is the first operator being added I think we can keep things relatively self contained rather than adding new directory structures

Reply:
I followed mostly the structure from tvm, like here: https://github.com/dmlc/tvm/tree/master/topi/include/topi.

Reply:
I don't think we need to follow the structure of tvm for the addition of a single operator